File: app/src/main/java/farasense/mobile/model/realm/FaraSenseSensorDaily.java
package farasense.mobile.model.realm;

import com.google.gson.annotations.SerializedName;

import java.util.Date;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class FaraSenseSensorDaily extends RealmObject {

    @SerializedName("id")
    Integer id;
    @SerializedName("timestamp")
    Date date;
    @SerializedName("total_kwh")
    Double totalKwh;
    @SerializedName("total_power")
    Double totalPower;
    @PrimaryKey
    @SerializedName("long_timestamp")
    Long timestamp;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public Double getTotalKwh() {
        return totalKwh;
    }

    public void setTotalKwh(Double totalKwh) {
        this.totalKwh = totalKwh;
    }

    public Double getTotalPower() {
        return totalPower;
    }

    public void setTotalPower(Double totalPower) {
        this.totalPower = totalPower;
    }

    public Long getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(Long timestamp) {
        this.timestamp = timestamp;
    }
}

File: app/src/main/java/old_farasense/model/realm/FaraSenseSensorDaily.kt
//package old_farasense.model.realm
//
//import com.google.gson.annotations.SerializedName
//import java.util.Date
//import io.realm.RealmObject
//import io.realm.annotations.PrimaryKey
//
//open class FaraSenseSensorDaily : RealmObject() {
//
//    @PrimaryKey
//    @SerializedName("long_timestamp")
//    var timestamp: Long? = null
//    @SerializedName("id")
//    var id: Int? = null
//    @SerializedName("timestamp")
//    lateinit var date: Date
//    @SerializedName("total_kwh")
//    var totalKwh: Double? = null
//    @SerializedName("total_power")
//    var totalPower: Double? = null
//}
